pkg/oidc/storage.go

```diff
@@ -539,11 +539,13 @@ func NewStorageFacade(storage Storage, rp rp.RelyingParty, privateKey *rsa.Priva
 	return &storageFacade{
 		Storage: storage,
 		signingKey: signingKey{
-			id:        "id",
+			id:        KeyID,
```

Comment:
Why? We create a constant that has the same value, even though it is not used elsewhere?